### DISCUSSION TITLE
[@types/semantic-release] branch type is missing on Context interface

### DISCUSSION THREAD

**Discussion by nitzano** [DefinitelyTyped/DefinitelyTyped#56405] [Issues with a @types package]
When trying to develop a semantic release plugin it looks like `branch` type is missing on `Context` interface from `@types/semantic-release`:

Context definition: 

https://semantic-release.gitbook.io/semantic-release/developer-guide/plugin#verifyconditions

```
branch
  Information on the current branch
  Object keys:
      channel
      tags
      type
      name
      range
      accept
      main
```

Context's type interface:

https://github.com/DefinitelyTyped/DefinitelyTyped/blob/e827f569f4ace3fbbf17fb1295fb7ab35a262f1b/types/semantic-release/index.d.ts#L374

I can open a PR which fixes it if needed

[a comment by an automated account is not shown]

**Reply by peterblazejewicz** [reply]
Yes, please do. Looks like a legit change, Im not familiar with plugins for this package, thx!

**Reply by nitzano** [reply]
Sure, will do
